src/i18n.ts

```diff
@@ -0,0 +1,62 @@
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+import LanguageDetector from 'i18next-browser-languagedetector'
+// import HttpBackend from 'i18next-http-backend' // *** added this ***
+import { BACKEND_URL } from './services/constants'
+
+i18n
+//   .use(HttpBackend) // *** added this ***
+  // detect user language
+  // learn more: https://github.com/i18next/i18next-browser-languageDetector
+  .use(LanguageDetector)
+  // pass the i18n instance to react-i18next.
+  .use(initReactI18next)
+  // init i18next
+  // for all options read: https://www.i18next.com/overview/configuration-options
+  .init({
+    debug: true,
+    fallbackLng: 'fr',
+    supportedLngs: ['en', 'es', 'fr'], // *** added this ***
```

Comment:
Comment node needed
Also are you sure "es" is available already?

Reply:
I 've remove "es"